src/popup/components/Main.vue

```diff
@@ -408,6 +439,65 @@ export default {
         });
     },
 
+    // More options
+    toggleMoreOptions(index) {
+      if (this.moreOptions.index !== -1) {
+        this.moreOptions.index =
+          this.moreOptions.index !== index
+            ? index // show more options for another alias
+            : -1; // hide more options
+      } else {
+        this.moreOptions.index = index;
+      }
+      this.moreOptions.loading = false;
+    },
+    handleClickDelete(index) {
+      this.$modal.show("dialog", {
+        title: `Delete ${this.aliasArray[index].email}`,
+        text: "Do you really want to delete this alias?",
```

Comment:
For UX purpose, can we add the alias address in the confirm message here so user is sure about their action they're going to take :) ?